### ISSUE TITLE
Runtime error when trying to import OAuth2 Provider models for token deletion

### ISSUE THREAD

**Issue by charlesdsmith** [jazzband/django-oauth-toolkit#719]
I've been trying for days to import the models from the library so I could routinely delete expired refresh tokens, but whenever I do I receive the following error: ` Model class oauth2_provider.models.Application doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`

I tried adding `oauth2_provider.models.Application` to my `INSTALLED_APPS` but to no avail.  

I've seen someone import these models for this reason already: https://stackoverflow.com/questions/31341115/delete-expired-access-tokens

My current settings.py
`INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    'api',
    'oauth2_provider',
    #'provider',
    #'provider.oauth2'
    'rest_framework',
    'rest_framework.authtoken',
    'graphene_django',
    'corsheaders',
]`

What could be causing this?

**Comment by IvanAnishchuk** [comment]
Tried lazy importing? ```from oauth2_provider.models import get_access_token_model, get_application_model
Application = get_application_model()
```

**Comment by Shivaank121** [comment]
Yes, I tried lazy importing as suggested by @IvanAnishchuk but it's still showing the error ;
LookupError: No installed app with label 'admin'.
